docs/pages/guides/config-plugins.md

```diff
@@ -610,7 +686,86 @@ Notice that `expo-location` uses `version: '11.0.0'`, and `react-native-maps` us
 ```
 
 For the most _stable_ experience, you should try to have no `UNVERSIONED` plugins in your project. This is because the `UNVERSIONED` plugin may not support the native code in your project.
-For instance, say you have an `UNVERSIONED` Facebook plugin in your project, if the Facebook native code or plugin has a breaking change, that will break the way your project ejects and cause it to error on build.
+For instance, say you have an `UNVERSIONED` Facebook plugin in your project, if the Facebook native code or plugin has a breaking change, that will break the way your project prebuilds and cause it to error on build.
+
+## Static Modification
+
+Plugins can technically be used to regex file changes into application code, but these modifications are dangerous, if the template changes over time then the regex becomes hard to predict (similarly, if the user modifies a file manually or uses a custom template). Here are some examples of files you shouldn't modify manually, and alternatives.
+
+### Android Gradle Files
+
+Gradle files are written in either groovy or kotlin, they are used to manage dependencies, versioning, and other settings in the Android app. Instead of modifying them directly with `withProjectBuildGradle`, `withAppBuildGradle`, or `withSettingsGradle`, utilize the static `gradle.properties` file.
```

Comment:
```suggestion
Gradle files are written in either Groovy or Kotlin. They are used to manage dependencies, versioning, and other settings in the Android app. Instead of modifying them directly with the `withProjectBuildGradle`, `withAppBuildGradle`, or `withSettingsGradle` mods, utilize the static `gradle.properties` file.
```